package.json

```diff
@@ -23,10 +23,9 @@
   "license": "(OFL-1.1 AND MIT)",
   "dependencies": {},
   "devDependencies": {
-    "@wikimedia/less-plugin-clean-css": "^1.5.2",
+    "less-plugin-clean-css": "^1.5.1",
```

Comment:
Package dependencies are inserted in lexical order by the `npm` CLI. That means that the next time a new package is added to the set, this line will be re-ordered to under the `less` dependency making an unrelated change at that point. Hence, it is better to keep it at line 29 in this change.

Reply:
Also, this should probably also have an accompanying change in `package-lock.json`